recipe/meta.yaml

```
@@ -11,11 +11,12 @@ source:
   sha256: 3771a65699caee0f5b97cedada74a52b9ba4061bd8ad150d0115defb291c4581
 
 build:
-  number: 0
+  number: 1
   script:
     # The tarball shouldn't have egg-info--this one has paths in SOURCES.txt
     # that break the build on windows.
-    - rm -rf src/pyppmd.egg-info
+    - rd src/pyppmd.egg-info  # [win]
```

It's complaining about a "bad switch" so try a Windows path:
```suggestion
    - rd src\pyppmd.egg-info  # [win]
```